examples/nokia/srlinux-services/2node-srl-ixr6-with-oc-services.pbtxt

```diff
@@ -10,7 +10,7 @@ nodes: {
     vendor: NOKIA
     model: "ixr6e"
     config:{
-        image: "ghcr.io/nokia/srlinux:22.6.4"
+        image: "ghcr.io/nokia/srlinux:22.11.2"
```

Comment:
just remove this line so it defaults to the :latest tag (also in the other nokia examples as well)